### DISCUSSION TITLE
Randomized Intervals

### DISCUSSION THREAD

**Discussion by Halogen-ic** [Prince25/StockAlertBot#126] [Requests]
I recently had an issue with NewEgg where they detected automated access from my IP and blocked access for 24 hours (since I also check BestBuy I can't use a proxy) I was wondering if the ability to have a random interval with user-provided range might solve this issue and if so would it be within reason to add.

Here's a picture of the error message I get when I try to access NewEgg:
![NewEgg Screenshot](https://user-images.githubusercontent.com/94069950/141161377-c557b236-e999-4ed7-8279-b141eeeadcc2.jpg)

Just in case it's of any use my current Interval is 10 seconds and 2 seconds in between items on each site.

Also, this is an awesome project!

**Comment by Prince25** [comment]
Hello. Sorry this happened to you. Yes, a random interval sounds like a good idea and I'll add it to the list. Thank you!
